src/front-end/src/helper-methods.js

```diff
@@ -2,6 +2,27 @@ import firebase from './firebase-init';
 
 const auth = firebase.auth();
 
+export function resendVerificationEmail(user) {
+  user
+    .sendEmailVerification()
+    .then(() => {
+      window.alert('A new verification email has successfully been sent!');
+      console.log('Success');
+    })
+    .catch((error) => {
+      console.log(error);
+      switch (error.code) {
+        case 'auth/too-many-requests':
+          window.alert(
+            'Please check the inbox and junk folder of your email before pressing the button again',
+          );
+          break;
+        default:
+          break;
```

Comment:
Did you use a switch instead of an if here in case in the future we want to add more error code handling? If not, I'd use a simple if statement here since this switch only has one case.

Reply:
Yea I made a switch just incase we come across other errors in the future so it's easier to address!